source/extensions/filters/http/cache/simple_http_cache/simple_http_cache.cc

```diff
@@ -294,7 +294,8 @@ class SimpleHttpCacheFactory : public HttpCacheFactory {
         envoy::extensions::cache::simple_http_cache::v3::SimpleHttpCacheConfig>();
   }
   // From HttpCacheFactory
-  HttpCache& getCache(const envoy::extensions::filters::http::cache::v3::CacheConfig&) override {
+  HttpCache& getCache(const envoy::extensions::filters::http::cache::v3::CacheConfig&,
+                      Server::Configuration::FactoryContext&) override {
```

Comment:
It looks like you are plumbing in the context but then dropping it on the floo

I assume a follow-up PR will depend on this? Maybe comment in the description?

Reply:
Fair point, I am building an extension in a separate codebase which relies on the async client factory, as well as the stats scope. I can add that in a comment.

Reply:
The context will be used for further extensions, not specifically for the in-memory cache. I can add a comment to the `HttpCacheFactory`